### PR TITLE
cmd: rephrase deployment create flag, error and  help text

### PR DESCRIPTION
**Pull request by karencfv**
## Description
In order to improve UX, this PR substitutes the word "idempotency token" for "request ID". The help text is also slightly changed to make it more understandable.

## Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (improves code quality but has no user-facing effect)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation

### CODE REVIEW

File: cmd/deployment/create.go [hunk 1]
```diff
@@ -162,7 +162,7 @@ func init() {
 	createCmd.Flags().String("name", "", "Optional name for the deployment")
 	createCmd.Flags().BoolP("track", "t", false, cmdutil.TrackFlagMessage)
 	createCmd.Flags().Bool("generate-payload", false, "Returns the deployment payload without actually creating the deployment resources")
-	createCmd.Flags().String("request-id", "", "Optional idempotency token - Can be found in the Stderr device when a previous deployment creation failed, for more information see the examples in the help command page")
+	createCmd.Flags().String("request-id", "", "Optional request ID - Can be found in the Stderr device when a previous deployment creation failed, for more information see the examples in the help command page")
```
Comment (alaudazzi) [inline cmd/deployment/create.go:165]:
```suggestion
	createCmd.Flags().String("request-id", "", "Optional request ID - Can be found in the Stderr device when a previous deployment creation failed. For more information see the examples in the help command page")
```

File: cmd/deployment/create_help.go [hunk 2]
```diff
@@ -68,12 +68,7 @@ $ ecctl deployment create --name my-deployment --size 1024 --track --generate-pa
 ## Create a deployment through the file definition.
 $ ecctl deployment create --file create_example.json --track
 
-## To retry a when the previous deployment creation failed:
-$ ecctl deployment create
-The deployment creation returned with an error, please use the displayed idempotency token
-to recreate the deployment resources
-Idempotency token: GMZPMRrcMYqHdmxjIQkHbdjnhPIeBElcwrHwzVlhGUSMXrEIzVXoBykSVRsKncNb
-unknown error (status 500)
+## To retry a deployment when the previous deployment creation failed, use the request ID provided in the previous command's error response:
```
Comment (alaudazzi) [inline cmd/deployment/create_help.go:71]:
```suggestion
## To retry a deployment when the previous deployment creation failed, use the request ID provided in the error response of the previous command:
```
If you accept this suggestion, change the other occurrences as well.

File: docs/ecctl_deployment_create.adoc [hunk 3]
```diff
@@ -41,7 +36,7 @@ $ ecctl deployment create --file create_example.json --request-id=GMZPMRrcMYqHdm
 ----
   -f, --file string         DeploymentCreateRequest file definition. See help for more information
   -h, --help                help for create
-      --request-id string   Optional idempotency token - Can be found in the Stderr device when a previous deployment creation failed, for more information see the examples in the help command page
+      --request-id string   Optional request ID - Can be found in the Stderr device when a previous deployment creation failed, for more information see the examples in the help command page
```
Comment (alaudazzi) [inline docs/ecctl_deployment_create.adoc:39]:
```suggestion
      --request-id string   Optional request ID - Can be found in the Stderr device when a previous deployment creation failed. For more information see the examples in the help command page
```

File: docs/ecctl_deployment_create.md [hunk 4]
```diff
@@ -37,7 +32,7 @@ $ ecctl deployment create --file create_example.json --request-id=GMZPMRrcMYqHdm
 ```
   -f, --file string         DeploymentCreateRequest file definition. See help for more information
   -h, --help                help for create
-      --request-id string   Optional idempotency token - Can be found in the Stderr device when a previous deployment creation failed, for more information see the examples in the help command page
+      --request-id string   Optional request ID - Can be found in the Stderr device when a previous deployment creation failed, for more information see the examples in the help command page
```
Comment (alaudazzi) [inline docs/ecctl_deployment_create.md:35]:
```suggestion
      --request-id string   Optional request ID - Can be found in the Stderr device when a previous deployment creation failed. For more information see the examples in the help command page
```

**Comment by karencfv** [comment]
Thanks for the review @alaudazzi :) I have addressed your comments.

**Pull request merged**